pkg/codegen/python/gen.go

```diff
@@ -65,10 +65,8 @@ type typeDetails struct {
 type imports codegen.StringSet
 
 // defaultMinPythonVersion is what we use as the minimum version field in generated
-// package metadata if the schema does not provide a value. This version corresponds
-// to the minimum supported version as listed in the reference documentation:
```

Comment:
Why remove this comment?

Reply:
Ah I was going to add a PR comment for this, but forgot to do it.

Our [docs](https://www.pulumi.com/docs/languages-sdks/python/) don't mention an explicit version anymore, but just link to Python's supported version page

> Pulumi supports writing your infrastructure as code in the Python language running on any [supported version](https://devguide.python.org/versions/#versions).

Reply:
Comment still feels valid to me. We might not give the value in the documentation but it is a pointer to what the minimum supported version is.

Reply:
Yeah, that's a valid point. I restored the comment.